lib/tasks/update_sitemap.rake

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+namespace :sitemap do
```

Comment:
🌟 

Are you using the Heroku scheduler thingy to do this weekly or something?

Reply:
yep thats the plan! originally I was going to do it nightly but thats probably overkill for something like this.